flexmeasures/data/models/user.py

```diff
@@ -46,6 +60,9 @@ class User(db.Model, UserMixin):
     # Faster token checking
     fs_uniquifier = Column(String(64), unique=True, nullable=False)
     timezone = Column(String(255), default="Europe/Amsterdam")
+    account_id = Column(Integer, db.ForeignKey("account.id"), nullable=False)
+
+    account = db.relationship("Account", backref=db.backref("users", lazy=True))
```

Comment:
As `account_id` has `nullable=False`, that seems impossible to achieve. I guess deleting an Account with Users linking to it would lead to an error.
We could add `, cascade="all, delete-orphan", passive_deletes=True` to this relationship.backref, if we want to automate deleting users. But I feel I'd rather not delete users automatically, actually. How do you feel about this?